packages/uicore/src/components/ScrollShadowContainer/ScrollShadowContainer.module.css

```diff
@@ -0,0 +1,39 @@
+.body {
+  --shadowTop: inset 0px 8px 5px -5px rgba(0, 0, 0, 0.7);
+  --shadowBottom: inset 0px -8px 5px -5px rgba(0, 0, 0, 0.7);
+  position: relative;
+  overflow: auto;
+}
+
+.body::before {
+  content: '';
+  position: absolute;
+  inset: 0;
+  pointer-events: none;
+  z-index: 2;
+}
+
+.body .bodyContent {
+  position: relative;
+  z-index: 1;
+  height: 100%;
+  overflow: scroll;
```

Comment:
can we use `auto` instead? `scroll` shows a scrollbar even when it's not needed.
test this with a wired mouse plugged in in MacOS or on windows.